modules/wmco-prerequisites.adoc

```diff
@@ -75,6 +75,8 @@ The following information details the supported platform versions, Windows Serve
 1. This installation type is only supported when the `platform: none` field is set in the `install-config.yaml` file during cluster installation.
 --
 
+//Is this table still correct?
```

Comment:
@EricPonvelle, is the preceding table all correct?

Reply:
@kalexand-rh the current WMCO prereqs are all correct for 4.9 :+1: